### PR TITLE
Implement page detail

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement page detail

Desktop
localhost_5173_detail_3

Mobile
localhost_5173_detail_3 (1)

### CODE REVIEW

File: src/components/Breadcrumb/breadcrumbs.module.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
   padding: 0;
   list-style: none;
   gap: 1.5rem;
+  margin: 2.5rem 0;
```
Comment (hungdo-agilityio) [inline src/components/Breadcrumb/breadcrumbs.module.css]: can we use `margin-inline`

File: src/pages/Details/detail.module.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.container {
+  margin: 0 auto;
```
Comment (thanhnguyen-agilityio) [inline src/pages/Details/detail.module.css]: this should be for all page?

File: src/pages/Details/index.tsx [hunk 3]
```diff
@@ -1,5 +1,161 @@
...
+  { label: productDetailData.title },
+];
+
+// example data end
+
 const Details = () => {
-  return <></>;
+  return (
+    <div>
+      <Header />
```
Comment (hungdo-agilityio) [inline src/pages/Details/index.tsx]: Do others pages have Header and Footer? If yes create a layout and put these in

File: src/pages/Details/index.tsx [hunk 4]
```diff
@@ -1,5 +1,161 @@
+import Header from '../../components/Header/Header';
+import ProductDetail from '../../components/ProductDetail/ProductDetail';
+import { Product } from '../../types/product';
+import image1 from '../../assets/images/detail-small.jpg';
+import image2 from '../../assets/images/small-t-shirt.jpg';
```
Comment (thanhnguyen-agilityio) [inline src/pages/Details/index.tsx]: use meaningful naming, like `smallTShirtImage`

File: src/pages/Details/detail.module.css (hunk 2 above)
Comment (thanhtrantruong47) [reply src/pages/Details/detail.module.css]: Fixed

File: src/components/Breadcrumb/breadcrumbs.module.css (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/Breadcrumb/breadcrumbs.module.css]: Fixed

File: src/pages/Details/index.tsx (hunk 3 above)
Comment (thanhtrantruong47) [reply src/pages/Details/index.tsx]: Fixed

File: src/pages/Details/index.tsx (hunk 4 above)
Comment (thanhtrantruong47) [reply src/pages/Details/index.tsx]: Fixed

**Pull request merged**